### PR TITLE
Update Config Schema & Add Notification template schema

### PR DESCRIPTION
**Pull request by snehal0904**
## Checklist

- [ ] The issues that this PR fixes/closes have been mentioned above.
- [ ] What this PR adds/changes/removes has been explained.

### CODE REVIEW

File: schemas/Template.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    "properties": {
+      "Template": {
+        "$ref": "#/definitions/Template"
+      }
+    },
+    "required": [
+      "Template"
+    ],
+    "title": "Template",
+    "definitions": {
```
Comment (coolbung) [inline schemas/Template.json:13]: Add templateId

File: schemas/Templatecontent.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    "title": "Templatecontent",
+    "definitions": {
+      "Templatecontent": {
+        "$id": "#/properties/Templatecontent",
+        "type": "object",
+        "title": "The Template content schema",
+        "description": "All Template content.",
+        "required": [],
+        "properties": {
+          "template_id": {
```
Comment (coolbung) [inline schemas/Templatecontent.json]: Use camel casing for all. Also, this should be templateContentId right ? And additionally templateId needed as a reference from the other schema

File: schemas/Templatecontent.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            "$id": "#/properties/subject",
+            "type": "string"
+          },
+          "body": {
+            "$id": "#/properties/body",
+            "type": "string"
+          },
+          "replacement_tags": {
+            "$id": "#/properties/replacement_tags",
+            "type": "string"
```
Comment (coolbung) [inline schemas/Templatecontent.json]: Is this correct ?

File: schemas/Templatecontent.json [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    "title": "Templatecontent",
+    "definitions": {
+      "Templatecontent": {
+        "$id": "#/properties/Templatecontent",
+        "type": "object",
+        "title": "The Template content schema",
+        "description": "All Template content.",
+        "required": [],
+        "properties": {
+          "templateId": {
```
Comment (coolbung) [inline schemas/Templatecontent.json:25]: In this schema won't we need both - templateId & templateContentId ?

Comment (snehal0904) [reply schemas/Templatecontent.json:25]: yes, will need it. Added.

File: schemas/Templatecontent.json (hunk 3 above)
Comment (snehal0904) [reply schemas/Templatecontent.json]: changed this to array. We will store array of objects

File: schemas/Notificationlog.json [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    ],
+    "title": "Notificationlog",
+    "definitions": {
+      "Notificationlog": {
+        "$id": "#/properties/Notificationlog",
+        "type": "object",
+        "title": "The Notification log content schema",
+        "description": "All Notification log.",
+        "required": [],
+        "properties": {
```
Comment (coolbung) [inline schemas/Notificationlog.json:20]: This will also need a unique ID say notificationId ?

**Pull request merged**